scripts/twister

```diff
@@ -903,6 +908,8 @@ def main():
     if options.show_footprint or options.compare_report or options.release:
         options.enable_size_report = True
 
+    logger.info("Running tests with seed %u", options.seed)
```

Comment:
Silence is golden still. Unless this is going to be used for something, it would be better to not print it. (it won't be used for most targets)

Reply:
The goal is that when a test fails you will have the seed to be able to replay. Though you're right, this can be an error log `if` the test failed.